sbnobj/Common/CRT/classes.h

```diff
@@ -5,10 +5,26 @@
 #include "sbnobj/Common/CRT/CRTHit.hh"
 #include "sbnobj/Common/CRT/CRTTrack.hh"
 #include "lardataobj/AnalysisBase/T0.h"
+#include "lardataobj/RecoBase/OpFlash.h"
 #include "sbnobj/Common/CRT/CRTTzero.hh"
+#include "sbnobj/Common/CRT/CRTPMTMatching.hh"
 #include "sbnobj/Common/CRT/CRTHit_Legacy.hh"
 #include "sbnobj/Common/CRT/CRTTrack_Legacy.hh"
 #include "sbnobj/Common/CRT/CRTTzero_Legacy.hh"
 #include <vector>
 #include <map>
 #include <utility>
+
+namespace{
```

Comment:
Is this really needed? Are the declarations in classes_def.xml not sufficient?

Reply:
thanks for your comment, @chilge. These extra declarations were not needed.

Reply:
what about the namespace {...} ?